non-msrv-tests/tests/ui/not-zeroize/comma.stderr

```diff
@@ -0,0 +1,11 @@
+error: unsupported trait syntax, expected one of expected one of Clone, Copy, Debug, Default, Eq, Hash, Ord, PartialEq, PartialOrd
+ --> tests/ui/not-zeroize/comma.rs:2:22
+  |
+2 | #[derive_where(Clone,; T)]
+  |                      ^
```

Comment:
Maybe allow trailing commas? As that is the rust default.

Reply:
We do allow trailing commas, but not when `;` comes afterwards. We might wanna again think about some better syntax, this is all so un-Rusty.

In any case, I can change that.

Reply:
Had to bump the MSRV to 1.34 too, because 1.33 doesn't support `feature(unrestricted_attribute_tokens)`, which allows things like semi-colons in there:
https://github.com/rust-lang/rust/blob/41301c3b2371365b753c2ad6a74528a38f3815ce/compiler/rustc_feature/src/accepted.rs#L295.

Reply:
Done.

Reply:
yeah the syntax is indeed unrusty af :D